.github/workflows/docs-links.yaml

```diff
@@ -1,8 +1,7 @@
 name: docs links on juju.is/docs
 
 on:
-  schedule:
-    - cron: "0 13 * * *"
+  workflow_dispatch:
```

Comment:
this should probably be `on: workflow_dispatch`, without a colon as you're not passing any inputs, but it's just me nitpicking (I was surprised for a second that this would work actually)

Reply:
Yeah I am being lazy as im undoing this when we are back.

Reply:
right, makes sense!